### ISSUE TITLE
Question: auto scale instance upgrade (TMOS)

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-aws-cloudformation#130]
## Do you already have an issue opened with F5 support?
No

## Description
Refering to the documentation: https://github.com/F5Networks/f5-aws-cloudformation/tree/master/supported/autoscale/ltm/via-lb/1nic/existing-stack/bigiq#restoring-or-upgrading-the-solution
a stack upgrade to upgrade TMOS (that means, existing instances have a config and run version v14.1.x, / new instances run version v15.1.x) is possible.
I've tried it and the new instance loads at some point the backup UCS successfully:
` Aug 17 18:15:15 ip-10-100-3-157.eu-west-1.compute.internal info install_ucs.pm[4365]: UCS installation success.
`
But then, autoscale gets in error
```
2020-08-17T16:18:02.904Z silly: [pid: 16494] [scripts/autoscale.js] Instance status: BECOMING_PRIMARY
2020-08-17T16:18:02.906Z error: [pid: 16494] [scripts/autoscale.js] autoscaling error: Currently becoming primary. Exiting.
```

Device is inoperative (mcpd down)

How is it supposed to work?

## Template
f5-aws-cloudformation/supported/autoscale/ltm/via-lb/1nic/existing-stack/bigiq/
v5.7.0

## Severity Level

Severity: 5

**Comment by andreykashcheev** [comment]
We just have published new templates v5.7.1. which includes the change intended to address this issue. 

Let me know if you have any questions/concerns

**Comment by shyawnkarim** [comment]
Closing.

Please reopen or create another issue if you are still having trouble.
